pyproject.toml

```diff
@@ -13,7 +13,7 @@ readme = { file = "README.md", content-type = "text/markdown" }
 authors = [
     { name = "NVIDIA Corporation" },
 ]
-license = { text = "Apache-2.0" }
+license = { text = "Apache 2.0" }
```

Comment:
The `-` was chosen to match [SPDX's short name]( https://spdx.org/licenses/Apache-2.0.html )

Reply:
Good point @jakirkham , should other RAPIDS projects change instead @raydouglass ? Preemptively switched to "Request changes" to prevent auto-merge until we can decide what's the right way forward.